src/OrchardCore/OrchardCore.ReCaptcha.Core/ActionFilters/Abuse/IDetectAbuse.cs

```diff
@@ -0,0 +1,19 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace OrchardCore.ReCaptcha.Core.ActionFilters.Abuse
+{
+    public interface IDetectAbuse
+    {
+        AbuseDetectResult DetectAbuse(HttpContext context);
```

Comment:
Shouldn't the implementation be responsible for looking up for the HttpContext if it needs it?